modules/osdk-updating-projects.adoc

```diff
@@ -48,7 +48,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
-        image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.11 <1>
+        image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.12 <1>
```

Comment:
I believe this is an ophaned module (it was broken up at some point), but changes look technically accurate to me! 😅 I'll double-check the module further after this PR and we'll probably remove it from the branch going forward.

Reply:
Thanks! Yeah I avoided things in the _unused_topics/ directory, but I didn't check that all of the modules were in use before updating.